### PR TITLE
Remove marketplace tour temporarily

### PR DESCRIPTION
**Pull request by kdevnel**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes https://github.com/Automattic/woocommerce.com/issues/17611

This PR temporarily disables the In-App Marketplace tour. 

The tour will be reconfigured at a later stage, but there is currently a bug that prevents the site option for marking that a tour is dismissed or completed from being saved. This means the tour, and its associated task, can't be dismissed. To prevent issues for now, we're disabling the tour altogether until the redesign can handle this bug.

Note that this PR does include some content changes for the tour since the decision to remove it was made after the redesign was started. Rather than remove those, it seemed logical to save work for later.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch locally and from `plugins/woocommerce-admin` run `pnpm run build`
2. Visit the WooCommerce “Home” screen – (`wp-env` URL: http://localhost:8888/wp-admin/admin.php?page=wc-admin)
3. Look for the "Things to do next" section and notice that the marketplace tour task isn't displayed
4. Visit the URL that would trigger the tour and see that no tour appears (`wp-env` URL: localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions&tutorial=true)

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

**Review by Dan-Q — APPROVED** [review]
Works a treat.

File: plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx [hunk 1]
```diff
@@ -57,9 +58,15 @@ const renderTabs = ( props: TabsProps ) => {
 	for ( const tabKey in tabs ) {
 		tabContent.push(
 			<Button
-				className={ classNames( 'woocommerce-marketplace__tab-button', {
-					'is-active': tabKey === selectedTab,
-				} ) }
+				className={ classNames(
+					'woocommerce-marketplace__tab-button',
+					`woocommerce-marketplace__tab-${ lowerCase(
+						tabs[ tabKey ]?.title
```
Comment (andfinally) [inline plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx]: Is there a strong reason to use the `title` here? If we just use `tabKey`, we won't need to use `lowerCase`.

**Review by andfinally — APPROVED** [review]
Looking good! Just one minor query.

**Review by raicem — APPROVED** [review]
Works as described!

File: plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx (hunk 1 above)
Comment (kdevnel) [reply plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx]: Oh, that's a great improvement. I'll change that before merging.

Comment (kdevnel) [reply plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx]: Amended for both the tab buttons and tab anchors in https://github.com/woocommerce/woocommerce/pull/39618/commits/5571456b11fa3885fcc09b88c0df12c9150dc7ae

**Review by andfinally — APPROVED** [review]
Thanks for the tweak! Looks good!

**Pull request merged**